camptocamp/ngeo
# Support external WMS in query tool

For GSGMF-1290.

This patch allows connected (imported) WMS layers to be queried.  It used to work before, but an extra validation about "layers being in range", which requires WFS to work, was preventing the regular WMS layers to work.

Also, this patch includes a note in the ngeo-query-panel that is now shown for each possible action: click, drawbox, drawpolygon.  This informs the user about limitations when using a box or polygon (i.e. only queryable layers that support WFS will return results).

## How to test

 * run a desktop app (I like the desktop_alt one)
 * remove all layers from the layer tree and switch base map to "blank"
 * use the "Import Layer" tool to the left:
  * choose "Swiss Topo WMS" server
  * search for "dégâts"
  * add the layer "Carte des dégâts aux forêts" to the map
  * click on a cluster of points on the map
  * --> results should be shown at the bottom
